src/entry.js

```diff
@@ -90,22 +90,21 @@ class Entry {
     }
 
     // Ensure `entry` follows the correct format
-    const e = {
+    const dagNode = {
       hash: null,
       id: entry.id,
       payload: entry.payload,
-      next: entry.next,
+      next: entry.next.map(cid => ({ '/': cid })),
```

Comment:
@satazor @haadcode @shamb0t  Might we be able to approach backwards compatibility here by first checking for the cid and then checking for `entry.next` as a fallback?

Reply:
Yes we can indeed do that. Will change!

Reply:
I think it’s worth creating a helper in utils/ just for that, what do you think? The helper would read and write dag nodes, with an argument indicating which properties need to be linked.

Reply:
Love it. Let's see how it looks!

Reply:
@aphelionz I've created `toIpldNode` and `fromIpldNode` in the utils. Can you check how this looks?